### ISSUE TITLE
SRFlow: Learning the Super-Resolution Space with Normalizing Flow 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#60]
# TL;DR
High resolution image generation using normalizing flows. Unlike GAN, which requires adjustment of multiple loss terms, it learns by optimizing only the log-likelihood. It also uses invertible transformations, which allows one-to-one correspondence with latent representations, so it can also use high-resolution latent variables to perform style transformations.
![Output Single SR Image](https://user-images.githubusercontent.com/24524018/91837811-7cfe4480-ec87-11ea-805c-088998b76a32.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2006.14200

# Submission Dates(yyyy/mm/dd)
2020/06/25

# Authors and institutions
Andreas Lugmayr, Martin Danelljan, Luc Van Gool, Radu Timofte
- Computer Vision Laboratory, ETH Zurich 

# Methods

# Results

# Comments
ECCV2020
